### DISCUSSION TITLE
Define what language to translate into?

### DISCUSSION THREAD

**Discussion by patrickmac110** [Purfview/whisper-standalone-win#113] [Q&A] [answered]
I see this command, but how do I actually specify what language I want to translate to, if not English?
 --task {transcribe,translate}
                        whether to perform X->X speech recognition ('transcribe') or X->English translation
                        ('translate') (default: transcribe)
                        
Specifically, I have English audio, and I want to translate into Hatian Creole. 
This was my best guess:
```
.\whisper-faster.exe "A:\Downloads\Audio Clip (2023-11-16_21-50-48-037).m4a"  --model=large -f=text --task=translate-X->ha
```

**Comment by Purfview** [comment, marked as answer]
You can't, it translates only to English.
